iocore/net/SSLUtils.cc

```diff
@@ -2159,7 +2123,7 @@ ssl_callback_session_ticket(SSL *ssl, unsigned char *keyname, unsigned char *iv,
   ssl_ticket_key_block *keyblock = NULL;
   if (cc == NULL || cc->keyblock == NULL) {
     // Try the default
-    keyblock = global_default_keyblock;
+    keyblock = params->default_global_keyblock;
```

Comment:
Not sure I'm following your used just once comment.  The keyblock is used only within this function.  As long as we have acquired the SSLConfig, the keyblock value it refers to will not disappear.  It seems that in most cases, the overlap on config reload will be at most one, but it could be multiple safely as long as we have a referece to the SSLConfig before we start and hold it the entire time we are using the keyblock value.